.gitignore

```diff
@@ -4,3 +4,6 @@ bin/
 .polyglot*
 .tycho-consumer-pom.xml
 pom.tycho
+.classpath
+.settings
+.project
```

Comment:
Please revert this addition. There files are relevant for Eclipse project.
```suggestion
```

Reply:
I will do, but they are not under version control. They should not be under version control. Just sayin'

Reply:
They are under version control. Why should they not be under version control? M2E mainly consists out of Eclipse Plugin projects that require at least a `.project` and `.classpath` file since they cannot be derived, like for example for Maven projects. Or do you have something else in mind?

Reply:
.classpath is not. should be ignored. :)

Reply:
Sorry, but no for Plugin projects `.classpath` should not be ignored.